client/src/components/CategoryView/CategoryView.js

```diff
@@ -16,7 +26,7 @@ function CategoryView() {
         </Col>
         <Col lg={12} className="addPlaceButton">
           <Button type="primary" icon={<PlusOutlined />} size="large">
-            Add Restaurant
+            Add {btnCategory}
```

Comment:
cant we use `{currentCategory.name}` here?

Reply:
@sassansh the category name is plural that's why I couldn't use it😖

Reply:
ahh fine! Good for now then. Well think of a solution later